yet-another-docker-plugin/src/main/resources/com/github/kostyasha/yad/launcher/DockerComputerJNLPLauncher/init.sh

```diff
@@ -42,18 +42,34 @@ fi
 
 cd "$JENKINS_HOME"
 
+if [ "$NO_CERTIFICATE_CHECK" = true ]
```

Comment:
well... that's usually the way I wrote the shell equality. But == is also accepted. So, no problem to add it.

and true vs 'true' ... it is the same. true is considered as a string. This said, it will remove potential misunderstanding. So fixing it.